### ISSUE TITLE
Document the usage of signed certificates.

### ISSUE THREAD

**Issue by lednersinn** [input-output-hk/jormungandr#1333]
As far as I understand in the recent versions the certificates which are posted to the blockchain derived only from public keys. (see the related documentation change at https://github.com/input-output-hk/jormungandr/commit/6ac59ee8035944df5ec0a1f394e66c28c909365f ) My question is: how do we register our signed certificates? Where do we prove that a node is really created by us?

**Comment by rdlrt** [comment]
When submitting transaction (w/cert), you need to do auth using your account secret key, check doco [here](https://github.com/input-output-hk/jormungandr/blob/master/doc/stake_pool/registering_stake_pool.md#submitting-to-a-node)

**Comment by lednersinn** [comment]
@rdirt, the transaction is signed by the person who pays for it, and that person is not necessarily the owner.
I am not asking about registering to Cardano's testnet stake pool registry where the owner and the transaction fee payer must match. I am asking about registering the stake pool on the blockchain, and a feasible route for stake pools with multiple owners.

**Comment by rdlrt** [comment]
Yes , the reply was for the same, certificate sign is only required if you would want to register your pool in genesis, else  the details of the change is in #1063 and relevant docos (relevant part highlighted in previous comment) seem up to date.
Now, with regards to multiple owners - not sure if thats ready to use yet, my understanding is it isnt.

**Comment by rinor** [comment]
@lednersinn do you still need clarification about this? Please let us know so we can provide more info.
